applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -26,22 +19,20 @@
 import reactor.util.annotation.NonNull;
 
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.EnumSet;
-import java.util.List;
+import java.util.Arrays;
 import java.util.Locale;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 /**
- * Client that contains all the operations for KeyValues in Azure Configuration Store.
+ * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration Store.
+ *
```

Comment:
In the fullness of time this client class is a prime location for a few paragraphs of useful text, code samples, etc. Add that to your TODO list (or to Alan's)

Reply:
Created an issue for this #3322 .